docs/css.md

```diff
@@ -1,115 +1,125 @@
 # Whitehall CSS Styleguide
 
-To be read in conjunction with the [GDS Way CSS styleguide](https://gds-way.cloudapps.digital/manuals/programming-languages/css.html)
+To be read in conjunction with the following:
 
-## File structure and scope
+- [GDS Way CSS styleguide](https://gds-way.cloudapps.digital/manuals/programming-languages/css.html)
+- [GOVUK Component Conventions](https://github.com/alphagov/govuk_publishing_components/blob/main/docs/component_conventions.md)
+
+## Namespacing
+
+### Components
```

Comment:
We should have something here to explain the difference between handling components in GOV.UK components and view components, as this seems an area to easily get confused with.

Reply:
Thanks - great point, I'll add something in

Reply:
Done